scripts/appleseed.package.py

```diff
@@ -190,6 +191,103 @@ def merge_tree(src, dst, symlinks=False, ignore=None):
         raise Error(errors)
 
 
+def create_mac_icon(image, icon_path):
```

Comment:
These functions are macOS-specific, I suggest to move them to the appropriate section.